cmd/virt-prerunner/main.go

```diff
@@ -109,6 +109,18 @@ func buildVMConfig(ctx context.Context, vm *virtv1alpha1.VirtualMachine) (*cloud
 					diskConfig.Path = fmt.Sprintf("/mnt/%s/cloud-init.iso", volume.Name)
 				case volume.ContainerRootfs != nil:
 					diskConfig.Path = fmt.Sprintf("/mnt/%s/rootfs.raw", volume.Name)
+				case volume.PersistentVolumeClaim != nil:
+					diskConfig.Path = fmt.Sprintf("/mnt/%s", volume.Name)
```

Comment:
I think we should do the same file stat thing to PVC too, since it can be a FS PVC or a raw block PVC